css-three-column-layout/index.html

```diff
@@ -0,0 +1,146 @@
+    </div>
+
+    <div class="row justify-content-center">
+
+      <div class="card">
+        <div class="row justify-content-center">
+          <div class="circle">
+            <img src="images/merge.png" alt="">
+          </div>
+        </div>
+        <div class="row justify-content-center"><h3>Streamlined QMS</h3></div>
+        <div class="row"><p>A QMS that includes modles for all stages of the product life cycle from Design Control to CAPA</p></div>
+
+      </div>
+
+      <div class="card">
+        <div class="row justify-content-center">
+          <div class="circle">
+            <img src="images/cloud-computing.png" alt="">
+          </div>
+        </div>
+        <div class="row justify-content-center"><h3>Integrates with existing tools</h3></div>
+        <div class="row"><p>Enzyme QMS integrates with your existing tools, such as JIRA, GitHub and Jenkins with Enzyme</p></div>
+
+      </div>
+
+      <div class="card">
+        <div class="row justify-content-center">
+          <div class="circle">
+            <img src="images/discussion.png" alt="">
+          </div>
+        </div>
+        <div class="row justify-content-center"><h3>In-depth Onboarding</h3></div>
+        <div class="row"><p>We train your entire team and provide you with support when and as it's needed</p></div>
+
+      </div>
+
+
+
+    </div>
+    <!--
+  <div class="circle"> -->
+
+
+  </div>
+
+  </body>
+
+
+
+
+
+
+
+
+
+
+
+
+<!-- <body>
+
+  <div class="container">
+
+    <div class="row">
+      <h2>Designed for life science. Optimized for SaMD and Digital Health.</h2>
+    </div>
+
+    <div class="row space-evenly">
+
+      <div class="thirds">
+
+        <div class="row circle">
+          <img src="images/cloud-computing.png" alt="cercle">
+        </div>
+
+        <div class="row">
+          <div class="full-column">
+            <h3>Streamlined QMS</h3>
+            <p>A QMS that includes modles for all stages of the product life cycle from Design Control to CAPA</p>
+          </div>
+        </div>
+
+      </div>
+
+      <div class="thirds">
+
+        <div class="row circle">
+
+          <div class="center">
+
+          </div>
+
+          <img src="images/cloud-computing.png" alt="cercle">
+        </div>
+
+        <div class="row">
+          <div class="full-column">
+            <h3>Integrates with existing tools</h3>
+            <p>Enzyme QMS integrates with your existing tools, such as JIRA, GitHub and Jenkins with Enzyme</p>
+          </div>
+        </div>
+
+      </div>
+
+      <div class="thirds">
+
+        <div class="row circle">
+          <img src="images/cloud-computing.png" alt="cercle">
+        </div>
+
+        <div class="row">
+          <div class="full-column">
+            <h3>In-depth Onboarding</h3>
+            <p>We train your entire team and provide you with support when and as it's needed</p>
+          </div>
+        </div>
+
+      </div>
+
+
+    </div>
+
+  </div>
+
+</body> -->
```

Comment:
Please remove commented code prior to submitting for review

Reply:
<img width="1680" alt="Screen Shot 2022-03-18 at 13 25 36" src="https://user-images.githubusercontent.com/99709530/159078450-9f6625f6-58e2-4593-81b7-7b45c2716980.png">